src/sknnr/transformers/_cca_transformer.py

```diff
@@ -23,15 +24,14 @@ def fit(self, X, y):
 
         X, y = np.asarray(X), np.asarray(y)
         self.cca_ = CCA(X, y)
+        self.set_components(self.cca_)
+        self.projector_ = self.cca_.projector(n_components=self.n_components_)
```

Comment:
We don't necessarily need to set an estimator attribute here, we could instead just do this in `transform` instead.  Preference?

Reply:
Agreed and fixed.  Now using `self.ordination_.projector(n_components=self.n_components_)` in `transform`.